### DISCUSSION TITLE
🕷 v3.10.x

### DISCUSSION THREAD

**Discussion by trexfeathers** [SciTools/iris#5570] [Releases]
<div align="center">

| Manager | Buddy | Milestone | Feature Branch | Project |
| :---: | :---: | :---: | :---: | :---: |
| @HGWright | @trexfeathers | [v3.10](https://github.com/SciTools/iris/milestone/50) | [v3.10.x](https://github.com/SciTools/iris/tree/v3.10.x) | - |

</div>

<div align="center">

| Minor Release | Status | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.10.0rc0` | [![status](https://img.shields.io/badge/v3.10.0rc0-released-green)](https://github.com/SciTools/iris/discussions/5570) | `30-07-2024` | [Docs](https://scitools-iris.readthedocs.io/en/v3.10.0rc0/index.html) |  [v3.10.0rc0](https://github.com/SciTools/iris/releases/tag/v3.10.0rc0) | [rc_iris](https://anaconda.org/conda-forge/iris?version=3.10.0rc0&channel=rc_iris) | [scitools-iris (--pre)](https://pypi.org/project/scitools-iris/3.10.0rc0/) |
| `v3.10.0` | [![status](https://img.shields.io/badge/v3.10.0-released-green)](https://github.com/SciTools/iris/discussions/5570) | `13-08-2024` | [Docs](https://scitools-iris.readthedocs.io/en/v3.10.0/index.html) | [v3.10.0](https://github.com/SciTools/iris/releases/tag/v3.10.0) | [main](https://anaconda.org/conda-forge/iris?version=3.10.0&channel=main) | [scitools-iris](https://pypi.org/project/scitools-iris/3.10.0/) |

</div>

**Comment by trexfeathers** [comment]
Hoping to make this a release focussed on mesh features 🙂

**Comment by trexfeathers** [comment]
`v3.10.0.rc0` has now been released 🚀

**Comment by HGWright** [comment]
`v3.10.0` has now been released! 🚀
